configuration/pih/htmlforms/hiv/section-hiv-plan.xml

```diff
@@ -940,9 +900,80 @@
                         </p>
                     </obsgroup>
                 </div>
-            </div>
-        </section>
-    </includeIf>
+            </includeIf>
+            <hr style="border: 1px dashed #ccc;"/>
+
+            <!-- Syphilis and STI drug order -->
+            <drugOrder>
+                <orderTemplate>
+                    <orderProperty name="action"/>
+                    <div style="padding:0px;">
+                        <orderProperty name="concept" label="pihcore.drugOrders.concept">
+                            <optionGroup concept="PIH:Syphilis and STI drug set"/>
+                        </orderProperty>
+                    </div>
+                    <div style="padding:0px;">
+                        <orderProperty name="drugNonCoded" label="Non-coded Formulation"/>
```

Comment:
You are including non-coded formulation but not actual drug formulations?  Is that intentional?